Scripts/Runtime/Entities/Lifecycle/Spawner/EntitySpawner.cs

```diff
@@ -157,16 +177,40 @@ public Entity SpawnWithPrototypeImmediate(TEntitySpawnDefinition spawnDefinition
         // SPAWN API - IN JOB
         //*************************************************************************************************************
 
-        public JobHandle AcquireEntitySpawnWriterAsync(out EntitySpawnWriter<TEntitySpawnDefinition> entitySpawnWriter)
+        public AccessControlledValue<EntitySpawnWriter<TEntitySpawnDefinition>>.AccessHandle AcquireWithSharedWriteHandle()
```

Comment:
Since these are public now, should we add some ///docs?

Reply:
yea, easy `///<inheritdoc`

Reply:
Docs added and refined the API
 - Remove spawn methods from the spawn system
 - Make `EntitySpawner.Schedule` internal